openapi.yaml

```diff
@@ -301,6 +301,10 @@ definitions:
           type: integer
         inclusionProof:
           $ref: '#/definitions/InclusionProof'
+        signature:
```

Comment:
we should probably be a bit more verbose with the name, `signedEntryTimestamp` maybe?

Reply:
I almost feel like log entry should be a type in itself with logIndex, body, and integrated time so it's clear log entry includes those three (and not e.g. inclusion proof)

Reply:
btw, do we want this to be replaced with an RFC 3161 signed timestamp over (logIndex, body) in the future?

Reply:
Probably not replaced by, but in addition to? I don't love the idea of promoting usage of the RFC3161 format outside of where we need it for compatibility.

Reply:
Oh yeah, definitely. the other option is a JSON based one, which would look roughly like a packaged signature field + message digest + cert chain

Reply:
I'm also in favor of shifting the inclusion proof and the signature into their own level of the LogEntry object. We should also probably add the log ID (in case we ever needed to rotate to a different log for performance / maintenance reasons)

perhaps something like:
```
LogEntry:
    type: object
    additionalProperties:
      type: object
      properties:
        logId:
          type: string
        logIndex:
          type: integer
          minimum: 0
        body:
          type: object
          additionalProperties: true
        integratedTime:
          type: integer
        verification:
          type: object
          properties:
            inclusionProof:
              $ref: '#/definitions/InclusionProof'
            signedEntryTimestamp:
              $ref: '#/definitions/SignedEntryTimestamp'
      required:
        - "logId"
        - "logIndex"
        - "body"
        - "integratedTime"
```

Reply:
Cool, I renamed it to `signedEntryTimestamp` and moved `signedEntryTimestamp` and `inclusionProof` under `verification`

I figure we can add in the logID in a separate PR so that this one doesn't get too big!